build.sc

```diff
@@ -28,6 +29,11 @@ trait CommonModule extends CrossUnRootedSbtModule with PublishModule {
     )
   )
 
+  override def repositories = super.repositories ++ Seq(
+    MavenRepository("https://oss.sonatype.org/content/repositories/snapshots"),
+    MavenRepository("https://oss.sonatype.org/content/repositories/releases")
+  )
```

Comment:
Why is this necessary?

Reply:
mill sanity check was failing until I put this in, could not find firrtl-interperter snapshot was first error. It could of been something else that changed I suppose